src/main.js

```diff
@@ -78,12 +77,59 @@ function switchScreens(closingWindow, openingWindow){
 
 function userInput() {
   currentPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
-  posterImage.src = currentPoster.imageURL;
-  posterTitle.innerHTML = currentPoster.title;
-  posterQuote.innerHTML = currentPoster.quote;
-  images.push(currentPoster.imageURL);
-  titles.push(currentPoster.title);
-  quotes.push(currentPoster.quote);
+  var error = errorMessages();
+  if (!error) {
+    posterImage.src = currentPoster.imageURL;
+    posterTitle.innerHTML = currentPoster.title;
+    posterQuote.innerHTML = currentPoster.quote;
+    images.push(currentPoster.imageURL);
+    titles.push(currentPoster.title);
+    quotes.push(currentPoster.quote);
+    switchScreens('.main-poster', '.poster-form');
+    clearForm();
+    removeAllErrors();
+  }
+}
+
+function errorMessages() {
+  var imageMsg = checkUserEntry(userPosterImage.value, imageError);
+  var titleMsg = checkUserEntry(userPosterTitle.value, titleError);
+  var quoteMsg = checkUserEntry(userPosterQuote.value, quoteError);
+  if (imageMsg || titleMsg || quoteMsg) {
+    return true;
+  } else {
+    return false; 
+  }
```

Comment:
Fun fact, you could refactor this using the values returned from `checkUserEntry` since they're already booleans:

```javascript
function errorMessages() {
  var imageMsg = checkUserEntry(userPosterImage.value, imageError);
  var titleMsg = checkUserEntry(userPosterTitle.value, titleError);
  var quoteMsg = checkUserEntry(userPosterQuote.value, quoteError);

  return imageMsg || titleMsg || quoteMsg;
}
```